dev/com.ibm.ws.logstash.collector/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -25,4 +25,7 @@ maxFieldLength=Maximum field length
 maxFieldLength.desc=Specifies maximum number of characters for the message field associated with the events for various sources. If you specify a value of 0 or -1, the maximum number of characters is unlimited.
 
 maxEvents=Maximum events per second
-maxEvents.desc=Specifies the maximum number of events to be sent per second. If you specify a value greater than zero, then it will be used for throttling the events.
\ No newline at end of file
+maxEvents.desc=Specifies the maximum number of events to be sent per second. If you specify a value greater than zero, then it will be used for throttling the events.
```

Comment:
A nit: Try to avoid using "per."  Suggested rewrite of the first sentence:

Specifies the maximum number of events sent in a second.